shared/templates/template_OVAL_file_permissions

```diff
@@ -13,7 +13,12 @@
       <criterion comment="Check file mode of {{{ FILEPATH }}}" test_ref="test_file_permissions{{{ FILEID }}}" />
     </criteria>
   </definition>
-  <unix:file_test check="all" check_existence="all_exist" comment="Testing mode of {{{ FILEPATH }}}" id="test_file_permissions{{{ FILEID }}}" version="1">
+  {{%- if FILE_REGEX -%}}
+    {{# When a regex doesn't match any files, and check_existence is all_exist, the test evaluates to fail. #}}
```

Comment:
We also have templates template_OVAL_file_owner and template_OVAL_file_groupowner. For example, in OpenShift benchmark there are 3 rules suing regex for /etc/cni/net.d/.* and 3 rules for /etc/origin/openvswitch/.*  When you do this change only for template_OVAL_file_permissions, and the files don't exist, in each group the permissions check will pass but the owner and group checks will fail. Is it a problem?

Reply:
I suggest to solve it this way: add a new parameter to the template eg. fail_if_not_present.